frontend/src/libs/utils.ts

```diff
@@ -187,3 +187,13 @@ export function guessActiveSession(sessions: Session[]): Session | null {
 
     return null;
 }
+
+export function getReadableAssignmentStatus(assignment: Pick<Assignment, "active_offer_status">) {
+    // TODO: confirm wording to be easier for the instructor
+    switch (assignment.active_offer_status) {
+        case "accepted":
+            return "Assigned";
+        default:
+            return capitalize(assignment.active_offer_status);
```

Comment:
This is a UX question of whether we need to change the status to be more readable here